edge-modules/Analyzer/ResponseStatus.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Analyzer
+{
+    using System;
+
+    public class ResponseStatus
+    {
+        public string ModuleId { get; set; }
+
+        public string StatusCode { get; set; }
+
+        public string ResultAsJson { get; set; }
+
+        // TODO: consider removing
+        public long SequenceNumber { get; set; }
```

Comment:
Response status is used by twin and direct-methods only, so can I remove this sequence number?